packages/dev/core/src/Rendering/prePassRenderer.ts

```diff
@@ -583,6 +613,37 @@ export class PrePassRenderer {
         return cfg;
     }
 
+    /**
+     * Retrieves an effect configuration by name
+     * @param name
+     * @returns
+     */
+    public getEffectConfiguration(name: string): PrePassEffectConfiguration | null {

```

Comment:
nit:` Nullable<PrePassEffectConfiguration>` to stick with what we do elsewhere in the code base.